core/src/test/java/co/cask/wrangler/steps/PipelineTest.java

```diff
@@ -101,22 +100,35 @@ public void testMaskSuffle() throws Exception {
 
   @Test
   public void testQuantizationRangeAndPattern() throws Exception {
-    RangeMap<Double, String> rangeMap = TreeRangeMap.create();
-    rangeMap.put(Range.closed(0.1, 0.9), "A");
-    rangeMap.put(Range.closed(2.0, 3.9), "B");
-    rangeMap.put(Range.closed(4.0, 5.9), "C");
-    String s = rangeMap.get(2.2);
+    TreeMap<Double, String> rangeMap = new TreeMap<>();
+    rangeMap.put(0.1, "A");
+    rangeMap.put(0.9, null);
+    rangeMap.put(2.0, "B");
+    rangeMap.put(3.9, null);
+    rangeMap.put(4.0, "C");
+    rangeMap.put(5.9, null);
+
```

Comment:
Do you need a test where the ranges overlap? I don't think the code prevents that. I am also not sure what the semantics should be in that case... and what RangeMap would have done?